docs/_docs/installation/kubernetes/generic-configuration.adoc

```diff
@@ -388,6 +388,20 @@ Note that we use the external IP address (LoadBalancer Ingress) of the service.
 include::{javaFile}[tags=connectThinClient, indent=0]
 ----
 
+== Partition Awareness
+
+include::../../includes/partition-awareness.adoc[]
+
+To enable the partition awareness feature within scaling Kubernetes enviroment one should start a client within the cluster and configure it with `KubernetesConnectionConfiguration`.
+In this case, a client can connect to every pod in a cluster.
+
+Note that client configuration must be in sync with Ignite node configuration.
```

Comment:
What does this sentence mean? What type of synchronization is assumed? Confusing.

Reply:
I mean that list of addresses in ClientConfiguration must be in sync with list of addresses of IgniteConfiguration.

Reply:
Agree, looks confusting. Remove this sentence at all.